src/main/java/uk/gov/hmcts/reform/pip/data/management/utils/JsonExtractor.java

```diff
@@ -22,6 +27,10 @@
 
     SearchConfiguration searchConfiguration;
 
+    JsonSchema schema;
+
+    final String SCHEMA_FILE_NAME = "schema3-draft.json";
```

Comment:
I think we should move the name of the file to config instead, and read it into here, so that it can change in the future without needing to update the code.

Reply:
done